### PR TITLE
👌 Improve feedback when specifying invalid environment

### PR DESCRIPTION
**Pull request by deifyed**
## Description

Added a check to see if the environment exists and some feedback if it didn't.

## Motivation and Context

#174 

## How Has This Been Tested?

`make test-short`

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have added tests to cover my changes.
- [x] All new and existing tests passed.
- [ ] I have updated the release notes (for the next release).

### CODE REVIEW

File: pkg/okctl/okctl.go [hunk 1]
```diff
@@ -51,6 +51,9 @@ import (
 	"github.com/oslokommune/okctl/pkg/storage"
 )
 
+// ErrorEnvironmentNotFound indicates that the requested environment isn't available
+var ErrorEnvironmentNotFound = errors.New("environment not found")
```
Comment (yngvark) [inline pkg/okctl/okctl.go:55]: Consider adding more info: "environment not found: dev".

Even better:

"environment not found: dev. Possible values are: prod, staging"

which I guess you can find from o.RepoState.Clusters like you did for apply cluster.

**Review by yngvark — CHANGES_REQUESTED** [review]
Nice getting this fixed.

File: pkg/okctl/okctl.go (hunk 1 above)
Comment (deifyed) [reply pkg/okctl/okctl.go:55]: Possible values are on line 50 in applyapplication.go

I'll add the specified invalid environment as well

Comment (deifyed) [reply pkg/okctl/okctl.go:55]: Right. This is done. Please re-review @yngvark.

Idiomatic Go error handling is not my strong suit, so please do come with tips if you have any

**Review by yngvark — APPROVED** [review]
Perfect!

**Pull request merged**